src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserReleaseRoleRepositoryTests.cs

```diff
@@ -1002,6 +1001,323 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
             }
         }
 
+        [Fact]
+        public async Task IsUserPrereleaseViewerOnLatestPreReleaseRelease()
+        {
+            var publication = new Publication();
+            var user = new User();
+
+            var olderRelease = new Release
+            {
+                ApprovalStatus = Approved,
+                Publication = publication,
+                ReleaseName = "2019",
+                TimePeriodCoverage = CalendarYear
+            };
+
+            var latestPublishedRelease = new Release
+            {
+                Id = Guid.NewGuid(),
+                ApprovalStatus = Approved,
+                Publication = publication,
+                Published = DateTime.UtcNow,
+                ReleaseName = "2020",
+                TimePeriodCoverage = CalendarYear,
+                Version = 0
+            };
+
+            var latestRelease = new Release
+            {
+                ApprovalStatus = Approved,
+                Publication = publication,
+                Published = null,
+                ReleaseName = "2020",
+                TimePeriodCoverage = CalendarYear,
+                PreviousVersionId = latestPublishedRelease.Id,
+                Version = 1
+            };
+
+            var latestReleaseOtherPublication = new Release
```

Comment:
On this test and in all the other prerelease tests just added I think it makes sense to give it a `ApprovalStatus = Approved` to all the instances of `latestReleaseOtherPublication`.

In the other tests that this class already had we don't care too much what status `latestReleaseOtherPublication` has because those methods under test are not concerned with the status of the release.